docs/project-ops.md

```diff
@@ -0,0 +1,377 @@
+* [Working with projects](#working-with-projects)
+* [Working with clusters](#working-with-clusters)
+
+
+## Prerequisites
+
+- You’re logged on to the Gardener Dashboard.
+- You’ve created a cluster and its status is operational.
+
+It's recommended that you get acquainted with the resources in the [Gardener API](https://gardener.cloud/api-reference/).
+
+
+## Downloading `kubeconfig` for remote project operations
+The `kubeconfig` for project operations is different from the one for cluster operations. It has a larger scope and allows a different set of operations that are applicable for a project administrator role, such as lifecycle control on clusters and managing project members.
+
+Depending on your goal, you create a service account suitable for automation and download its `kubeconfig`, or you can get a user-specific `kubeconfig`. The difference is the identity on behalf of which the operations are performed. 
+
+### Download `kubeconfig` for a user
+
+Kubernetes doesn’t offer an own resource type for human users that access the API server. Instead, you either have to manage unique user strings, or use an OpenID-Connect (OIDC) compatible Identity Provider (IDP) to do the job. 
+
+Once the latter is set up, each Gardener user can use the `kubelogin` plugin for `kubectl` to authenticate against the API server:
+
+1. Set up `kubelogin` if you don't have it yet. More information: [kubelogin setup](https://github.com/int128/kubelogin#setup).
+
+1. Open the menu at the top right of the screen, then choose **MY ACCOUNT**.
+
+    ![Show account details](images/Show-account-details.png)
+
+1. On the **Access** card, choose the arrow to see all options for the personalized command-line interface access.
+   
+    ![Show details of OICD login](images/Show-details-of-OICD-login.png)
+
+    > The personal bearer token that is also offered here only provides access for a limited amount of time for one time operations, for example, in `curl` commands. The `kubeconfig` provided for the personalized access is used by `kubelogin` to grant access to the Gardener API for the user permanently by using a refresh token.
+
+1. Check that the right **Project** is chosen and keep the settings otherwise. Download the `kubeconfig` file and add its path to the `KUBECONFIG` environment variable.
+
+You can now execute `kubectl` commands on the garden cluster using the identity of your user.
+
+### Download `kubeconfig` for a Service Account
+
+1. Go to a service account and choose **Download**. 
+
+    ![Download service account kubeconfig](images/Download-service-account-kubeconfig.png)
+
+1. Add the downloaded  `kubeconfig` to your configuration.
+
+You can now execute `kubectl` commands on the garden cluster using the anonymous service account.
+
+## List Gardener API resources
+
+1. Using a `kubeconfig` for project operations, you can  list the Gardner API resources using the following command:
+   
+    ```
+    kubectl api-resources | grep garden
```

Comment:
regarding `kubectl` commands in this doument: For me, this looks like a duplication of Gardener documentation. It has nothing to do with the dashboard. If things change here, we need to adopt or might not be even aware of it. Why not link to the Gardener docu?

Reply:
@grolu: What exactly do you mean by "Gardener documentation" ? Documentation on [gardner.cloud](https://gardener.cloud/documentation/home/)?

The idea is to _move_ the source of this documentation from the Gardener documentation (https://gardener.cloud/documentation/home/) to one of the 'dev repos' of github.com/gardener. Georgi then wants to fetch this Markdown source and transform it into what you currently see in the Gardener documentation. He presented it a while ago in one of the KubeTech sessions, but the idea is also provided in the [README of hist docforge repository](https://github.com/gardener/docforge/blob/master/README.md).

@g-pavlov

Reply:
Hi @JensHaley - sorry for the late reply. By "Gardener documentation" I mean documentation for the gardener component (https://github.com/gardener/gardener). I understand that we move documentation to the dev repos, but when we put things lie shoot yamls or kubectl commands into our docu, we implicitly have the responsibility to keep things up-to-date that are not under our control.
However, I agreed with my fellow Dashboard team members that we - for now - merge the docu as is. We need to plan and dedicate time into this and update the documentation so that it meets our standards. By this, I do not mean that we don't recognize the effort you put into this. I think it is a good starting point but we need to move things that are not our responsibility to repos of the respective owners. To achieve this, we need to better unterstand docforge and how links between different documentation sources can be managed.
So let's merge the documentation and we will improve in iterations.